### PR TITLE
Add change_url feature

### PR DESCRIPTION
**Pull request by JimboJoe**
## Problem
- *URL for this application can't be changed simply after installation*

## Solution
- *Add `change_url` script*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : Maniack C
- [x] **Approval (LGTM)** : Maniack C
- [x] **Approval (LGTM)** : Kay0u
- **CI succeeded** : 
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/nextcloud_ynh%20enh_change_url%20(Official)/)
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by maniackcrudelis** [comment]
Code review ok.
Nice to have that feature, that's quite a shame for such a nice app to not already have it.
If I have time, I'll add it to each Official apps

File: scripts/change_url [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+# Define a function to execute commands with `occ`
+exec_occ() {
+  (cd "$final_path" && exec_as "$app" \
+      php occ --no-interaction --no-ansi "$@")
+}
+
+if [ $change_domain -eq 1 ]
+then
+	# Change the trusted domain
+	exec_occ config:system:set trusted_domains 1 --value=${new_domain}
```
Comment (kay0u) [inline scripts/change_url]: Is it possible to "untrust" the old domain before?

Like here: https://github.com/nextcloud/nextcloud-snap/issues/352#issuecomment-329530660

Comment (JimboJoe) [reply scripts/change_url]: Well, the trusted_domains is an ordered list, and the app domain is installed at position 1 [during install](https://github.com/YunoHost-Apps/nextcloud_ynh/blob/master/conf/config_install.json#L6), so we just have to replace the domain at this position.

**Comment by JimboJoe** [comment]
Will be merged in 3 days.

**Pull request merged**